src/c_api.rs

```diff
@@ -0,0 +1,612 @@
+use std::ffi::CString;
+use std::os::raw::{c_double, c_int, c_long};
+use std::ptr::null_mut;
+use std::str::FromStr;
+use std::{
+    ffi::CStr,
+    os::raw::{c_char, c_void},
+};
+
+use crate::redisjson::Format;
+use crate::{redisjson::RedisJSON, REDIS_JSON_TYPE};
+use redis_module::key::RedisKeyWritable;
+use redis_module::logging::log_notice;
+use redis_module::Context;
+use redis_module::{raw as rawmod, RedisError};
+use serde_json::Value;
+
+// extern crate readies_wd40;
+// use crate::readies_wd40::{BB, _BB, getenv};
+
+//
+// structs
+//
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct RedisModuleCtx {
+    _unused: [u8; 0],
+}
```

Comment:
This is duplicate code from `redismodule-rs`, this block can be removed.
```suggestion
```

Reply:
👍🏼